src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -241,9 +255,13 @@ function BaseValidateCodeForm(props) {
                         hasError={hasError}
                         autoFocus
                     />
+                    {hasError && <FormHelpMessage message={ErrorUtils.getLatestErrorMessage(props.account)} />}
```

Comment:
Any reason why we are moving the error here?

Reply:
Because otherwise it displays below the timer which looks a bit odd. This way it aligns with the other error you get if you try and sign in without entering a code.